### PR TITLE
Fix some docs examples

### PR DESCRIPTION
**Pull request by migueldvb**
## PR Summary

Fix some docs and provide information about git commands in the reference sheet.

## PR Checklist

- [ ] Code passes flake8 checker
- [ ] New features are documented, with docstrings and narrative docs
- [ ] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

File: doc/source/analyzing/analysis_modules/absorption_spectrum.rst [hunk 1]
```diff
@@ -1,12 +1,5 @@
 .. _absorption_spectrum:
 
-.. note::
```
Comment (matthewturk) [inline doc/source/analyzing/analysis_modules/absorption_spectrum.rst]: Do we want to remove this?

File: doc/cheatsheet.tex [hunk 2]
```diff
@@ -313,23 +313,29 @@ \subsection{Parallel Analysis}
 (such as halos or clumps).\\
 
 
-\subsection{Mercurial}
+\subsection{Git}
```
Comment (matthewturk) [inline doc/cheatsheet.tex]: Thanks for updating this!

File: doc/source/analyzing/analysis_modules/light_ray_generator.rst [hunk 3]
```diff
@@ -1,12 +1,5 @@
 .. _light-ray-generator:
 
-.. note::
```
Comment (matthewturk) [inline doc/source/analyzing/analysis_modules/light_ray_generator.rst]: Same here -- why remove this?

File: doc/source/quickstart/4)_Data_Objects_and_Time_Series.ipynb [hunk 4]
```diff
@@ -150,7 +150,7 @@
     "pylab.semilogx(zs, mass, '-xb')\n",
     "pylab.xlabel(\"Redshift\")\n",
     "pylab.ylabel(\"Mass in halos / Total mass\")\n",
-    "pylab.xlim(max(zs), min(zs))\n",
+    "pylab.xlim(right=max(zs))\n",
```
Comment (matthewturk) [inline doc/source/quickstart/4)_Data_Objects_and_Time_Series.ipynb]: I didn't know you could do this!

**Comment by matthewturk** [comment]
Also, @migueldvb , thank you!

File: doc/source/analyzing/analysis_modules/absorption_spectrum.rst (hunk 1 above)
Comment (migueldvb) [reply doc/source/analyzing/analysis_modules/absorption_spectrum.rst]: The warning notes are repeated on top and below the section header at http://yt-project.org/docs/dev/analyzing/analysis_modules/absorption_spectrum.html and http://yt-project.org/docs/dev/analyzing/analysis_modules/light_ray_generator.html . It seems that they both got there around the same in different commits. Here only the ones on top are removed.

**Comment by migueldvb** [comment]
Interesting! I made a tiny change in the last commit and the checks are running again. Let's see if it was a temporary glitch. Otherwise I don't know where the problem could be introduced.

File: doc/source/analyzing/analysis_modules/absorption_spectrum.rst (hunk 1 above)
Comment (matthewturk) [reply doc/source/analyzing/analysis_modules/absorption_spectrum.rst]: Awesome, thank you!

**Comment by Xarthisius** [comment]
@yt-fido test this please.

**Comment by matthewturk** [comment]
The remaining error is just this: https://github.com/yt-project/unyt/issues/101  .  I'm not sure if we should hold up on merging for that, or go ahead.

**Comment by matthewturk** [comment]
@jzuhone what do you recommend?

**Comment by jzuhone** [comment]
I would not worry about that now, go ahead and merge.

**Comment by munkm** [comment]
Thank you so much for these documentation updates, @migueldvb!  🎉 🥇 🎈

**Pull request merged**